### DISCUSSION TITLE
Testing Before First Release

### DISCUSSION THREAD

**Discussion by gregtkogut** [triboelectric/statick-action#10] [General]
This acts as an informal peer review before putting the action into official release status.

I wanted to stress test both the action and indirectly statick itself.

Here are a lists to the action results in my personal repo area.  I will update this list as I add more repos.  Please suggest more repos.  In addition, suggest different ways to exercise the action and statick, e.g. different profiles or settings.,

1. [Linux](https://github.com/gregtkogut/linux/actions)

@denewiler, @xydesa

**Comment by tdenewiler** [comment]
@xydesa and I have used the NTRTSim repository to test in the past as it has a variety of file types and a fairly large number of files. I have a fork running the Statick action.

https://github.com/tdenewiler/NTRTsim/actions

**Comment by gregtkogut** [comment]
@xydesa, @tdenewiler  - I've dropped the ball on this, but am testing a new release to 0.8.2 to make a full release.  

But on the statick-action self test, I'm running into an issue on the discovery plugin for the documentation profile.

 https://github.com/gregtkogut/statick-action/runs/6951548512?check_suite_focus=true

Does this repo need the same update to the discovery plugins as the main repo?

**Reply by gregtkogut** [reply]
Update:  This is not related to recent changes to the tool plugin code, but to older changes that changed the syntax for the "inherits_from" capability.  I have a PR to statick-md that (I think) fixes the issue, and also a question about fixing that syntax across all statick repos.

**Comment by gregtkogut** [comment]
I tested on four forked repos today:

https://github.com/gregtkogut/rclpy/actions
https://github.com/gregtkogut/rclcpp/actions
https://github.com/gregtkogut/ros2_documentation/actions
https://github.com/gregtkogut/kubeapps/actions

It appears to work...in addition to continuing to work for statick proper.  I didn't test a bunch of the myriad configuration options.  Are there any that should be tested, or are you guys OK with making the first proper full release of the action? @tdenewiler , @xydesa

**Comment by tdenewiler** [comment]
This is working for me on one of my favored test repositories. I tested with and without the `--check` flag and it works as expected. This is in good shape for a release. We may want to do a Statick proper release (`0.9.0`) this week, so we can sync up those release activities offline.

https://github.com/tdenewiler/NTRTsim/actions
